diff --git a/node_modules/ansi-color/lib/ansi-color.js b/node_modules/ansi-color/lib/ansi-color.js
index 1062c87..e7f908e 100644
--- a/node_modules/ansi-color/lib/ansi-color.js
+++ b/node_modules/ansi-color/lib/ansi-color.js
@@ -26,14 +26,14 @@ var ANSI_CODES = {
   "white_bg": 47
 };
 
-exports.set = function(str, color) {
-  if(!color) return str;
+exports.set = function (str, color) {
+  if (!color) return str;
 
   var color_attrs = color.split("+");
   var ansi_str = "";
-  for(var i=0, attr; attr = color_attrs[i]; i++) {
-    ansi_str += "\033[" + ANSI_CODES[attr] + "m";
+  for (var i = 0, attr; attr = color_attrs[i]; i++) {
+    ansi_str += "\x1b[" + ANSI_CODES[attr] + "m";
   }
-  ansi_str += str + "\033[" + ANSI_CODES["off"] + "m";
+  ansi_str += str + "\x1b[" + ANSI_CODES["off"] + "m";
   return ansi_str;
 };
\ No newline at end of file
